packages/twenty-front/src/modules/command-menu/components/CommandMenuContextChipGroupsWithRecordSelection.tsx

```diff
@@ -43,6 +46,12 @@ export const CommandMenuContextChipGroupsWithRecordSelection = ({
             totalCount,
           ),
           Icons: Avatars,
+          onClick:
+            contextChips.length > 0
+              ? () => {
+                  openRootCommandMenu();
+                }
```

Comment:
For straightforwardness: 

```suggestion
? openRootCommandMenu
```